src/resdk/collection_tables.py

```diff
@@ -0,0 +1,274 @@
+""".. Ignore pydocstyle D400.
+
+=================
+Collection Tables
+=================
+
+.. autoclass:: CollectionTables
+    :members:
+
+    .. automethod:: __init__
+"""
+import os
+from datetime import datetime
+from functools import lru_cache
+from io import BytesIO
+from typing import Callable, Dict, List, NamedTuple, Optional
+from urllib.parse import urljoin
+
+import pandas as pd
+import pytz
+
+from resdk import Resolwe
+from resdk.resources import Collection
+from resdk.utils.table_cache import default_cache_dir, load_cache, save_cache
+
+TPM = "tpm"
+COUNTS = "rc"
+EXPRESSION_TYPES = [TPM, COUNTS]
+
+META = "meta"
+
+CHUNK_SIZE = 1000
+
+
+class _TableSample(NamedTuple):
+    sample_slug: str
+    expression_files: Dict[str, str]
+    metadata: dict
+
+
+class CollectionTables:
+    """A helper class to fetch given collection's expression and meta data in a tables format.
```

Comment:
Line length of docstrings should [be kept below 72 characters](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).